.yarnrc.yml

```diff
@@ -0,0 +1,15 @@
+packageExtensions:
+  react-error-boundary@*:
+    dependencies:
+      prop-types: "*"
+  eslint-module-utils@*:
+    dependencies:
+      eslint-import-resolver-node: "*"
+  debug@*:
+    peerDependencies:
+      supports-color: "*"
```

Comment:
Yarn applies this one by default https://github.com/yarnpkg/berry/blob/6618f693adff26152d8e400c9dcc98b0f8ff00cc/packages/plugin-compat/sources/extensions.ts#L50-L55